cognite/neat/rules/importers/_cdf_classic.py

```diff
@@ -0,0 +1,231 @@
+            for type_ in types_
+        ]
+        rules.properties.append(
+            InformationInputProperty(
+                class_="SourceSystem",
+                property_="source",
+                value_type="string",
+                transformation=",".join(transformation),
+            ),
+        )
+
+    def _add_dataset(self, rules: InformationInputRules) -> None:
+        from cognite.neat.graph.extractors import DataSetExtractor
+
+        if not self._store.queries.has_type(CLASSIC_CDF_NAMESPACE[DataSetExtractor._default_rdf_type]):
+            return
+
+        rules.classes.append(
+            InformationInputClass(
+                class_="DataSet",
+                parent=None,
+            ),
+        )
+        rules.properties.extend(
+            [
+                InformationInputProperty("DataSet", "name", "string"),
+                InformationInputProperty("DataSet", "description", "string"),
+                InformationInputProperty("DataSet", "external_id", "string"),
+                InformationInputProperty("DataSet", "write_protected", "boolean"),
+                InformationInputProperty("DataSet", "metadata", "json"),
+            ]
+        )
+
+    def _add_asset(self, rules: InformationInputRules) -> None:
+        from cognite.neat.graph.extractors import AssetsExtractor
+
+        if not self._store.queries.has_type(CLASSIC_CDF_NAMESPACE[AssetsExtractor._default_rdf_type]):
+            return
+
+        rules.classes.append(
+            InformationInputClass(
+                class_="Asset",
+                parent="cdf_cdm:CogniteAsset",
+            ),
+        )
+
+        rules.properties.extend(
+            [
+                InformationInputProperty("Asset", "name", "string"),
+                InformationInputProperty("Asset", "external_id", "string"),
+                InformationInputProperty(
+                    "Asset",
+                    "parent_id",
+                    "Asset",
+                    transformation="classic_cdf:Asset(parent_id) -> classic_cdf:Asset(parent)",
+                ),
+                InformationInputProperty("Asset", "description", "string"),
+                InformationInputProperty(
+                    "Asset",
+                    "dataSet",
+                    "DataSet",
+                    transformation="classic_cdf:Asset(dataSet_id) -> classic_cdf:Asset(dataSet)",
+                ),
+                InformationInputProperty("Asset", "metadata", "json"),
+                InformationInputProperty("Asset", "source", "SourceSystem"),
+                InformationInputProperty(
+                    "Asset",
+                    "tags",
+                    "string",
+                    max_count=10,
+                    transformation="classic_cdf:Asset(labels) -> classic_cdf:Asset(tags)",
+                ),
+            ]
+        )
+        # Todo Deal with GeoLocation
```

Comment:
We also need to look up all relationships starting with an Asset, and figure out how it should be added.